index.html

```diff
@@ -1412,6 +1412,123 @@ <h3>Biometrics</h3>
     </section>
   </section>
 
+  <section class="informative">
+    <h2>Environmental and Ethical Considerations</h2>
+
+    <p>
+      The following section details certain key areas of focus when implementing a 
+      new DID menthod, or implementing a solution that incorporates DIDs.
+    </p>
+
+    <p>
+      A reliable guide for making assessments of various technologies and weighing
+      ethical considerations is the 
+      <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">W3C TAG Ethical Web Principles</a>
+      document.
+    </p>
+
+    <section class="informative">
+      <h3>Broad Ethical Principles</h3>
+      <p>
+        As with all things, strong consideration for the appropriate and ethical use
+        of technology should be made when implementing items related to DIDs.
+      </p>
+
+      <p>
+        As noted in the <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#principles">Principles</a>
+        section of the Ethical Web Principles, there are certain key goals that should 
+        apply to all Web standards and technologies. DIDs explicitly support several
+        of these goals, especially the following:
+        <ul>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy">Privacy</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#control">Individual Control</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#multi">Device Independence</a></li>
+        </ul>
+      </p>
+    </section>
+
+    <section class="informative">
+      <h3>Energy Usage and Environmental Impacts</h3>
+      <p>
+        When implementing or utilizing a DID method, consideration should be given
+        to the environmental impacts of any underlying technologies.  
+        Avoidance of excess energy use that does not further other equally important 
+        goals such as human rights or personal privacy, particularly when that energy 
+        is sourced from non-sustainable energy creation methods, should be a best 
+        practice that serves the interest of all.
+      </p>
+      <p>
+        Direct comparison of each factor, and the balances between them, should be
+        considered when evaluating particular methods or approaches. Notably, the
+        ability to misuse or control DIDs that could impede privacy might be
+        found to be inversely correlated with additional energy consumption.
+      </p>
+      <p>
+        The guiding principle that 
+        <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#sustainable">the
+        web must be an environmentally sustainable platform</a> should be followed.
+      </p>
```

Comment:
```suggestion
      <p>
        Implementers should keep in mind the guiding principle that 
        <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#sustainable">the
        web must be an environmentally sustainable platform</a>, but ultimately security and 
        protection of critical systems, like identifier networks that may become the foundation 
        for critical activities in human life, are the most important factors to account for.
      </p>
```

Reply:
`are the most important factors to account for` is a rather strong and subjective statement, which does not flow from nor integrate well with the quoted W3 principles.

I suggest a larger revision here --
s/
```
        web must be an environmentally sustainable platform</a>, but ultimately security and 
        protection of critical systems, like identifier networks that may become the foundation 
        for critical activities in human life, are the most important factors to account for.
      </p>
```
/
```
        web must be an environmentally sustainable platform</a>, and balance this with
        the security and protection of critical systems, like identifier networks that
        may become the foundation for critical activities in human life.
      </p>
```
/